reproducible/ubuntu/images_release.yaml

```diff
@@ -0,0 +1,6 @@
+steps:
+  - name: 'gcr.io/cloud-builders/bazel'
+    args: ['run', '//reproducible/ubuntu:${_IMAGE}']
```

Comment:
I think these substitutions are the old envsubst style. Cloud build supports some now that are built in. What is _IMAGE supposed to be here?

Reply:
IMAGE is the bazel-rule to run which shares a name with the image to be created.  I created this one cloudbuild rule and then I added the var in each of the cloud build triggers. So for the ubuntu_16_0_4_release_vX.Y.Z trigger -> _IMAGE=ubuntu_16_0_4
ubuntu_16_0_4_build_release_vX.Y.Z trigger -> _IMAGE=ubuntu_16_0_4_build
node_8_5_0_ubuntu_release_vX.Y.Z trigger -> _IMAGE=node_8_5_0_ubuntu
node_8_5_0_ubuntu_build_release_vX.Y.Z trigger -> _IMAGE=node_8_5_0_ubuntu_build

this way we can just have one cloudbuild.yaml.  We can also split it up.  I use one of the built in variables for the tagging, `${COMMIT_SHA}`